### PR TITLE
Migrating to CI Runners AX-591

### PR DESCRIPTION
**Pull request by ola-rozenfeld**
https://linear.app/engflow/issue/AX-591/migrate-debian-11-jobs-to-ci-runners

Prerequisite: https://github.com/EngFlow/prod/pull/13635

### CODE REVIEW

File: .bazelrc
Comment (quval) [inline .bazelrc:1]: Unrelated?

**Review by quval — APPROVED** [review]

**Review by anfelbar — APPROVED** [review]

File: .bazelrc
Comment (quval) [reply .bazelrc:1]: (Discussed offline)

**Review by quval — APPROVED** [review]
